docs/tutorial.md

```diff
@@ -262,7 +262,7 @@ git appraise show 1e6eb14c8014
 Once a review has been accepted, you can merge it with the tool:
 
 ```shell
-git appraise submit --merge 1e6eb14c8014
+git review merge 1e6eb14c8014
```

Comment:
"Once a review has been accepted, you can merge it with the tool: `git appraise submit --merge 1e6eb14c8014`"
vs
"Once a review has been accepted, you can merge it with the tool: `git review merge 1e6eb14c8014`"